docs/architecture-decisions/0005-implement-an-accessible-navigation-system.md

```diff
@@ -0,0 +1,202 @@
+challenging (if not impossible) for users of assistive technologies to separate
+their interactions with the menu link and the submenu toggle.
+
+The current release of SmartMenus (1.2.1) does not support placing this toggle
+element outside of the link element, so it will be necessary to replace it with
+a library which does support this behavior.
+
+
+### Option 1: SmartMenus 2
+
+The maintainer for SmartMenus is aware of this problem in the library, as we and
+others have reported it via GitHub (see [issue 204](https://github.com/vadikom/smartmenus/issues/204) and [issue 231](https://github.com/vadikom/smartmenus/issues/231)). He has
+committed to including this as an improvement in the future release of
+SmartMenus 2, and the documentation already available [includes this among its available
+features](https://configurator.smartmenus.org/).
+
+However, at the time of writing [SmartMenus 2 is only available via one alpha
+release](https://github.com/vadikom/smartmenus/releases) - and the maintainer has [cautioned us against using it in a production
+system yet](https://github.com/vadikom/smartmenus/issues/245).
+
+* Pros: Just upgrading our existing menu library as it improves over time is a
+  good philosophy to follow.
+
+* Cons: The timeline for the release of SmartMenus 2 is not clear, and cannot
+  be guaranteed to happen. Additionally, the significant changes to the library
+  with version 2 will likely require a significant level of effort - so the
+  benefits of a simple upgrade may not accrue based on the materials published
+  so far.
+
+
+### Option 2: Accessible Menu
+
+One alternative to SmartMenus is the [Accessible Menu](https://accessible-menu.dev/) library, which takes as its
+starting point the [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/) for navigation menus. One of
+the supported patterns is the [Disclosure Navigation Menu with Top-Level Links](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation-hybrid/),
+which includes the needed separation between navigation links and submenu
+toggles.
+
+While the provided example includes this separation only at the top level of the
+menu, the pattern can be extended to all levels - [as demonstrated in this
+CodePen](https://codepen.io/matt-bernhardt/pen/poBRKrY).
+
+#### Benefits of Accessible Menu
+
+This menu, when properly configured, appears to implement all the best practices
+for an accessible navigation menu for web application.
+
+#### Drawbacks of Accessible Menu
+
+There are several significant drawbacks to using this menu library.
+
+* The most significant drawback is that it imposes some operating costs on users
+  of assistive technology that are not ideal. Our recent accessibility testing
+  [included some details about this problem](https://github.mit.edu/Accessibility-FY2024/omeka/issues/4#issuecomment-182190) while we discussed this menu issue.
```

Comment:
I don't have access to this link. Please either move the content to a location we all have access to (maybe our own Jira ticket that tracks that GitHub issue if we have one) or summarize the important aspects here.

Reply:
I'm re-working this section with a summary and direct quotation from Rich - but would like to still like to the issue queue for transparency's sake, even if not everyone has access to the link. Would you be okay with that?

Reply:
Not really. I think either the core team should have access to supporting links or we should move the content to some place we can access it (like in our own tracking Jira ticket, etc). Linking to something nobody but you can access feels like the opposite of transparency :)

Reply:
I agree that links should go someplace we all have access to - so it makes sense to me that we get you and Adam access to the review issues (which goes along with what we talked about yesterday, that we should just generally have access to repos like that, rather than only project team members).

Because we're having this discussion about a library we're deciding not to implement, I don't know that there will be a Jira ticket that could get this information - the implementation ticket will be focused more on describing the current work, rather than work we decided not to do.

If the re-wording I did this afternoon doesn't go far enough in your eyes, I'm happy to expand the quoted material here, or put together a summary in Confluence of the problem space. I'll also get you and Adam access to the DAS repo.